ruby2.7/cookiecutter-aws-sam-hello-ruby/cookiecutter.json

```diff
@@ -3,5 +3,8 @@
     "runtime": "ruby2.7",
     "architectures": {
         "value": []
-    }
+    },
```

Comment:
Unrelated to this PR but what do we normally do with deprecated runtime templates?

Reply:
We just remove the entire directory